canvas_sdk/v1/data/condition.py

```diff
@@ -5,10 +5,14 @@
 from canvas_sdk.v1.data.user import CanvasUser
 
 
-class ConditionQuerySet(ValueSetLookupQuerySet):
-    """ConditionQuerySet."""
+class ClinicalStatus(models.TextChoices):
```

Comment:
Noting that changes to this file and to the patient file necessitate a documentation PR.